server/crawler/starlark_docstring_parser.go

```diff
@@ -101,5 +102,22 @@ func parseType(rawType string) *StarlarkArgumentType {
 			}
 		}
 	}
+
+	if listTypeRegexp.MatchString(trimmedType) {
+		nestedTypeGroups := listTypeRegexp.FindStringSubmatch(trimmedType)
+		// nolint: gomnd
+		if len(nestedTypeGroups) >= 2 {
+			parsedInnerType1, err := StarlarkValueTypeString(nestedTypeGroups[1])
+			if err != nil {
+				// unknown type, unable to parse it
+				return nil
```

Comment:
@gbouv this may be hard to change, but is this not an error that the user should be informed of?

Reply:
Ideally, but here the consumer of this information is the user of the package, which might not be the author of the package. We should inform the author of the package when it writes it, not when a user consumes it. So, not entirely sure what we can do at the indexer level here. It should be something we validate when we lint the package or something like this
IDK, if you have an idea let me know, happy to make things better